src/containers/Page/components/Footer.tsx

```diff
@@ -101,32 +108,64 @@ const FooterWrapper = () => {
     },
   ];
 
+  const ZendeskWrapper = styled.div`
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+  `;
+
+  const StyledZendesk = styled(ZendeskButton)`
+    position: relative;
+    top: 20px;
```

Comment:
Knappen er ca 40px så dette er kun halvparten for å plassere den på skillet.

Reply:
Kanskje legge igjen en kommentar på dette?